lib/firmata.js

```diff
@@ -1411,6 +1451,148 @@ Board.prototype.stepperStep = function(deviceNum, direction, steps, speed, accel
 };
 
 /**
+ * Asks the Arduino to configure a hardware or serial port.
+ * @param {number} port The serial port to use (HW_SERIAL1, HW_SERIAL2, HW_SERIAL3, SW_SERIAL0,
+ * SW_SERIAL1, SW_SERIAL2, SW_SERIAL3)
+ * @param {number} baud The baud rate of the serial port.
+ * @param {number} bytesToRead The number of bytes to read on each iteration of the main loop
+ * (not yet used)
+ * @param {number} rxPin [SoftwareSerial only] The RX pin of the SoftwareSerial instance
+ * @param {number} txPin [SoftwareSerial only] The TX pin of the SoftwareSerial instance
+ */
+
+Board.prototype.serialConfig = function(portId, baud, bytesToRead, rxPin, txPin) {
```

Comment:
Need this to be a single object with each of those args as a property—that'll give the API enough flexibility to work with other platforms (I had to do the same to i2cConfig)